### ISSUE TITLE
Warning on start up "Warning (lsp-mode): Unknown notification: @/tailwindCSS/projectInitialized"

### ISSUE THREAD

**Issue by vjohansen** [merrickluo/lsp-tailwindcss#70]
When I load a tsx file initially i get  a "*Warnings*" buffer with 

` ■  Warning (lsp-mode): Unknown notification: @/tailwindCSS/projectInitialized`

If I call lsp-restart-workspace I get

```
 ■  Warning (lsp-mode): Unknown notification: @/tailwindCSS/projectsDestroyed
 ■  Warning (lsp-mode): Unknown notification: @/tailwindCSS/projectInitialized
```

lsp-version :  LSP :: lsp-mode 20240418.201, Emacs 29.3.50, gnu/linux

My guess is than it can be easily fixed by added a notification handler to the make-lsp-client call

See for example lsp-csharp.el which uses

```
                  :notification-handlers (ht ("o#/projectadded" 'ignore)
                                             ("o#/projectchanged" 'ignore)
                                             ("o#/projectremoved" 'ignore)
                ...
```

**Comment by morhaham** [comment]
please fix

**Comment by merrickluo** [comment]
should be fixed in #73
